indico/modules/events/timetable/templates/pdf/timetable.css

```diff
@@ -0,0 +1,424 @@
+  margin: 0;
+}
+
+.contrib-timeline-content > :is(p, div) b {
+  line-height: 2em;
+}
+
+:is(.contrib-timeline-content, .sub-contrib-timeline) > :is(p, div):not(:last-child) {
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.description {
+  break-inside: avoid;
+}
+
+:is(.contrib-timeline-content, .sub-contrib-timeline) .description > *:last-child {
+  margin: 0;
+  padding: 1px 10px;
+  border-radius: 3px;
+  background-color: rgba(0, 0, 0, 0.1);
+  word-break: break-all;
+}
+
+.sub-contrib-timeline {
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  border-radius: 3px;
+  padding: 10px;
+}
+
+.sub-contrib-timeline > li > p > b {
+  font-style: italic;
+}
+
+.sub-contrib-timeline > li {
+  padding: 5px;
+  background-color: white;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  border-bottom-width: 3px;
+  border-radius: 3px;
+  display: block;
+  break-inside: avoid;
+}
+
+.sub-contrib-timeline > li:not(:last-child) {
+  margin-bottom: 5px;
+}
+
+.sub-contrib-timeline > li > h4 {
+  margin: 0;
+  font-size: 1em;
+  font-style: normal;
+}
+
+.sub-contrib-timeline .contrib-timeline-content > :is(p, div):not(:last-child) {
+  margin: 0;
+  margin-bottom: 5px;
+}
+
+.pagebreak.pagebreak-children > .day-entry:not(:last-of-type) {
+  break-after: always;
+}
+
+.session-block-date {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 20px 0 10px 5px;
+  break-after: avoid;
+}
+
+.session-block-date > .timebox {
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  margin: 0;
+}
+
+/* Day */
+
+.day-entry > section {
+  /* break-inside: avoid; */
+  padding-left: 36px;
+  border-left: 3px solid rgba(0, 0, 0, 0.1);
+  margin-left: 36px;
+}
+
+/* .day-wrapper {
+  break-inside: avoid;
+} */
+
+/* Content box */
+.session-block-content-box {
+  border: 2px solid rgba(0, 0, 0, 0.1);
+  border-top: 0;
+}
+
+.session-block-content-box:last-child {
+  border-radius: 0 0 3px 3px;
+}
+
+.session-block-content-box.description {
+  margin-top: 0;
+  border-top: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  break-inside: avoid;
+}
+
+.session-block-content-box.description li > * {
+  margin: 0;
+  break-inside: avoid;
+}
+
+.session-block-content-box.description li > *:last-child {
+  font-size: 0.8em;
+  line-height: 1.5em;
+  margin: 5px 0 0 0;
+  opacity: 0.8;
+}
+
+/* Time */
+
+.day-entry {
+  position: relative;
+  display: block;
+  margin-bottom: 10px;
+}
+
+.time-text {
+  color: rgba(0, 0, 0, 0.5);
+  width: 75px;
+  text-align: center;
+  line-height: 1.4em;
+  font-size: 1em;
+  background-color: white;
+}
+
+.time-from {
+  position: absolute;
+  top: 0;
+}
+
+.time-to {
+  margin-top: -1.4em;
+  break-before: avoid;
+}
+
+.contrib-heading {
+  break-inside: avoid;
+}
+
+.contrib-heading > :is(h4, span) {
+  display: inline-block;
+  margin: 0;
+}
+
+.contrib-heading > span {
+  margin-right: 5px;
+}
+
+.contrib-heading > h4 {
+  margin-right: 5px;
+  line-height: 1.4em;
+}
+
+.bold {
+  font-weight: bold;
+}
```

Comment:
Why a custom class and not `<strong>` / `<b>`?

Reply:
Made it all use `<b>...</b>`